plugins/agent.py

```diff
@@ -62,8 +62,8 @@ def do_GET(self):
 
         query = parse_qs(urlparse(self.path).query)
         command = query['command'][0].split(' ')
-        async = bool(int(query.get('async', [False])[0]))
-        output = run_command(command, asynchronous=async)
+        asyn = bool(int(query.get('asyn', [False])[0]))
```

Comment:
would this change the input syntax? maybe we should leave keywords inside of query the same?
similar for event below